sda-commons-web-autoconfigure/src/main/resources/org/sdase/commons/spring/boot/web/logging/logback-json.xml

```diff
@@ -1,14 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Removing this file makes it a breaking change, doesn’t it?

Reply:
Yes.

1. Should be ok, because we are still before v1.
2. With your latest investigations, the file may come back.

I'll put this to draft again.

Reply:
As far as I know, Baymax is using this in prodcution. If it is actually removed we should give them a heads-up.

Reply:
Not removed any more.